tests/test_enterprise/api/test_views.py

```diff
@@ -4977,3 +4977,90 @@ def test_put_method_not_allowed(self):
         self.assertEqual(response.status_code, 405)
         response = response.json()
         self.assertEqual(response['detail'], 'Method "PUT" not allowed.')
+
+
+class TestEnterpriseUserLinkView(BaseTestEnterpriseAPIViews):
+    """
+    Test `EnterpriseUserLinkView`
+    """
+    ENTERPRISE_USER_LINK_ENDPOINT = 'enterprise-user-link'
+    USERNAME = "unlinkedtestuser"
+
+    def setUp(self):
+        super().setUp()
+        # unlinked_user = factories.UserFactory(
+        #     username=TEST_USERNAME,
+        #     is_active=True,
+        #     is_staff=False,
+        # )
+        # unlinked_user.set_password(TEST_PASSWORD)
+        # unlinked_user.save()
+
+        enterprise_customer = factories.EnterpriseCustomerFactory()
+        enterprise_customer_invite_key = factories.EnterpriseCustomerInviteKeyFactory(
+            enterprise_customer=enterprise_customer,
+            expiration_date=datetime.utcnow() + timedelta(days=365)
+        )
+        invalid_enterprise_customer_invite_key = factories.EnterpriseCustomerInviteKeyFactory(
+            enterprise_customer=enterprise_customer,
+            expiration_date=datetime.utcnow() - timedelta(days=365)
+        )
+
+        self.enterprise_customer = enterprise_customer
+        self.enterprise_customer_invite_key = enterprise_customer_invite_key
+        self.invalid_enterprise_customer_invite_key = invalid_enterprise_customer_invite_key
```

Comment:
You can actually drop these and assign the values returned by the factories directly to `self. enterprise_customer` etc above